modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/colorful-head/index.css

```
@@ -0,0 +1,8 @@
+.fragment_238 {
```

Replace `fragment_238` to `colorful-head`.

```suggestion
.colorful-head {
```

This is a simple change, just to match to pattern used in others fragments